wicket-core/src/test/java/org/apache/wicket/ajax/AjaxRequestHandlerTest.java

```diff
@@ -306,6 +312,94 @@ void ajaxRedirectSetsNoCachingHeaders()
 		assertEquals("no-cache, no-store", tester.getLastResponse().getHeader("Cache-Control"));
 	}
 
+	private static class Wicket7074Application extends MockApplication
+	{
+
+		@Override
+		public Class<? extends Page> getHomePage()
+		{
+			return Wicket7074.class;
+		}
+
+
+		@Override
+		public RuntimeConfigurationType getConfigurationType() {
+			// this ise needed so that no filters are added
```

Comment:
```suggestion
			// this is needed so that no filters are added
```

Reply:
Can we use more stable way to make sure that there are no filters ? E.g. code that removes all registered filters.
Otherwise if later a filter is added to the default ones in DEPLOYMENT mode then this test will silently change its behavior and do nothing.

Reply:
@martin-g This is true but 

![image](https://github.com/apache/wicket/assets/462655/888c755f-399f-4231-a3f1-218ef968a492)

Thus the only way I see is add a clearResponseFilters method. Or do you have a better idea?

Reply:
https://github.com/apache/wicket/pull/655/commits/a5d7c973fbf1f906ea1b6ab81cdde02c4cdf3ffc#r1332582655